hack/build-integration-kubernetes.sh

```diff
@@ -104,8 +104,10 @@ main(){
 
   # Hack to get go into kind control plane
   exec::nerdctl rm -f go-kind 2>/dev/null || true
-  exec::nerdctl run -d --name go-kind golang:"$GO_VERSION" sleep Inf
+  exec::nerdctl pull --quiet golang:"$GO_VERSION"
+  exec::nerdctl run -d --pull never --name go-kind golang:"$GO_VERSION" sleep Inf
```

Comment:
I think `nerdctl run` now has `--quiet` too

Reply:
Oh. Let me check.

Reply:
You are right. Will amend.

Reply:
Done.

Reply:
Probably you forgot to push the amended commit